bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -99,6 +99,11 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
                uint32_t,
                4,
                "Parameter used to heuristically compute the 'optimal' batch size");
+  CONFIG_PARAM(batchingIncreaseFactor, std::string, "0.1", "The increase/decrease rate");
```

Comment:
Add word "adaptive" to new parameters to simplify a look for them. 
Cut other words in the names (increase->inc, condition->cond, etc).

Reply:
Done, thanks.